WordPressCom-Stats-iOS/Services/WPStatsServiceRemote.m

```diff
@@ -1067,16 +1067,12 @@ - (NSURLSessionDataTask *)operationForCountryForDate:(NSDate *)date
                 countryName = [countryInfoDict[key] stringForKey:@"country_full"];
             }
             StatsItem *statsItem = [StatsItem new];
-            statsItem.label = countryName;
+            statsItem.label = [NSString stringWithFormat:@"%@ %@",
```

Comment:
I believe that just applies the localized date/number formatters to arguments, but won't do anything about text direction. Bidirectional Text is complicated and painful, but the gist of it is that UIKit will decide on the label direction by looking at the first characters of the string.